README.md

```diff
@@ -172,6 +172,21 @@ repeating the mistakes of the past.
     - For services, checkout our [best practices](https://mobify.atlassian.net/wiki/display/SYS/Service+Best+Practices).
     - For front-end web apps, checkout our [best practices](https://mobify.atlassian.net/wiki/questions/85917798/what-are-our-best-practices-for-building-front-end-web-application).
 
+### Use checklists
+
+There is a reason NASA makes extensive use of checklists - memories are faulty and unreliable.
+
+Checklists are a great way to ensure:
```

Comment:
I think one of the biggest things that checklists help with is **consistency**.  That's reflected indirectly in the points below, but I think it's worth having it's own bullet point.